UI/ClientUI.cpp

```diff
@@ -651,6 +656,11 @@ ClientUI::ClientUI() :
 
     // Set the root path for image tags in rich text.
     GG::ImageBlock::SetDefaultImagePath(ArtDir().string());
+
+    // Set facet to format timestamp in chat.
+    boost::posix_time::time_facet* facet = new boost::posix_time::time_facet();
+    facet->format(ClientUI::DisplayTimestampFormat().c_str());
+    m_date_format_sstream.imbue(std::locale(std::locale(""), facet));
```

Comment:
does this call to `imbue` take ownership of `facet`?

Reply:
Yes, it takes:
http://www.cplusplus.com/reference/locale/locale/locale/
> The constructed locale object takes over responsibility for deleting this facet object.